src/pysolid/grid.py

```diff
@@ -80,12 +80,11 @@ def calc_solid_earth_tides_grid(dt_obj, atr, step_size=1e3, display=False, verbo
 
     # resample to the input size
     if num_step > 1:
+        in_shape = tide_e.shape
         out_shape = (int(atr['LENGTH']), int(atr['WIDTH']))
         vprint('PYSOLID: resize data to the shape of {} using order-1 spline interpolation'.format(out_shape))
-        kwargs = dict(order=1, mode='edge', anti_aliasing=True, preserve_range=True)
-        tide_e = resize(tide_e, out_shape, **kwargs)
-        tide_n = resize(tide_n, out_shape, **kwargs)
-        tide_u = resize(tide_u, out_shape, **kwargs)
+        kwargs = dict(order=1, mode="nearest", grid_mode=True)
+        ndi.zoom(np.stack([tide_e, tide_n, tide_u]), [1, *np.divide(out_shape, in_shape)], **kwargs)
```

Comment:
The resized `tide_e/n/u` are not returned. Could you fix this?

Reply:
I am curious about the syntax of `[1, *np.divide(out_shape, in_shape)]`, it works nicely. Is there a document on it?

Reply:
this is a form of unpacking arguments: https://docs.python.org/3/tutorial/controlflow.html#unpacking-argument-lists https://treyhunner.com/2018/10/asterisks-in-python-what-they-are-and-how-to-use-them/
like how you can call a function `f(a, b, c)` with 
```python
later_args = (2, 3)
f('blah', *later_args)
```
you can make a list with `['blah', *later_args]`.

If it seems "too clever", i can break it out to two lines

Reply:
and thanks for catching that!

Reply:
Thanks for the explanation, I love it!